components/centraldashboard/app/api_contributors.ts

```diff
@@ -32,13 +98,30 @@ const EMAIL_RGX = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
 const surfaceProfileControllerErrors = (info: {res: Response, msg: string, err: any}) => {
     const {res, msg, err} = info;
     const code = (err.response && err.response.statusCode) || 400;
-    const error = err.body || msg;
-    console.error(`${msg} ${error}`);
-    apiError({res, code, error});
+    const devError = err.body || '';
+    // Msg is the developer reason of what happened, devError is the technical details as to why
+    console.error(msg+(devError?` ${devError}`:''));
+    apiError({res, code, error: devError || msg});
 };
 
 export class ContributorAPI {
```

Comment:
*nit* This class should probably be named `ContributorApi` to be consistent with Google TS naming guidelines.

Reply:
Done